notebooks/en/rag_llamaindex_librarian.ipynb

```diff
@@ -72,13 +72,74 @@
     "    llama-index-llms-ollama"
```

Comment:
<p>"Run Ollama service in the background."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>